rsciio/trivista/_api.py

```diff
@@ -0,0 +1,745 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2022 The HyperSpy developers
+#
+# This file is part of RosettaSciIO.
+#
+# RosettaSciIO is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# RosettaSciIO is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with RosettaSciIO. If not, see <https://www.gnu.org/licenses/#GPL>.
+
+import xml.etree.ElementTree as ET
```

Comment:
Do we care about XML vulnerabilities? I don't know much about them but maybe adding a note somewhere about this?

https://docs.python.org/3/library/xml.html#xml-vulnerabilities

(Well, just realized that many other readers make use of this module too...)